presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSecurityConfigProvider.java

```diff
@@ -0,0 +1,263 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import org.apache.kafka.common.config.SslConfigs;
+
+import java.util.Properties;
+
+/**
+ *  {@KafkaSecurityConfig} manages Kafka configuration related to connection security. I.e. if the security protocol
+ *  'SSL' is being used. If so, additional properties might be supplied, i.e. location and password of the truststore
+ *  and keystore to be used.
+ */
+public class KafkaSecurityConfigProvider
+{
+    private Properties props;
+
+    private String sslProtocol;
+    private String sslProvider;
+    private String sslCipherSuites;
+    private String sslEnabledProtocols;
+    private String sslKeystoreType;
+    private String sslKeystoreLocation;
+    private String sslKeystorePassword;
+    private String sslKeyPassword;
+    private String sslTruststoreType;
+    private String sslTruststoreLocation;
+    private String sslTruststorePassword;
+    private String sslKeymanagerAlgorithm;
+    private String sslTrustmanagerAlgorithm;
+    private String sslEndpointIdentificationAlgorithm;
+    private String sslSecureRandomImplementation;
+
+    public KafkaSecurityConfigProvider()
+    {
+        // no-args constructor
+    }
+
+    @Config("kafka." + SslConfigs.SSL_PROTOCOL_CONFIG)
```

Comment:
I am not a big fan of referencing constants in external library for Presto config keys. 
We want those to be stable. i.e. not change if we update Kafka library version. Please inline those.

Reply:
Also the convention in presto is to use `.` for separating logical sections in key and `-` for separating words.
So e.g we should have:
`kafka.ssl.cipher-suites` not `kafka.ssl.cipher.suites`.

Possibly we should also drop `.ssl` in most of the config properties.

Reply:
Inlining names - will do.

regarding property names - those are derived from the underlying kafka-clients lib, if they should be adjusted to fit presto convention, then i could also get rid of the ssl prefix as well.

Reply:
inlined and adjusted property names